Dockerfile

```diff
@@ -5,13 +5,16 @@ ARG LIBXEXT_VERSION=2:1.3.4-*
 FROM nvidia/cuda:11.7.1-devel-ubuntu20.04 AS tm_base_env
 ARG LIBXRENDER_VERSION
 ARG LIBXEXT_VERSION
+ARG CUDA_ARCH
+
+ENV CMAKE_ARGS="-DCUDA_ARCH:STRING=${CUDA_ARCH}"
```

Comment:
To save time when building with a different cuda arch, probably desirable to move this down to where the actual GPU code gets built. Somewhere like https://github.com/proteneer/timemachine/blob/master/Dockerfile#L106

Otherwise this is just in the base environment and you would have to rebuild OpenMM just to change the cuda arch